### DISCUSSION TITLE
dbgenerated date is using my machine local time

### DISCUSSION THREAD

**Discussion by randomprogramming** [prisma/prisma#13767] [Q&A]
Hello, I have two dates which are generated automatically like this: 

```
verificationTokenValidUntil DateTime? @default(dbgenerated("NOW() + interval '1 hour'"))
createdAt                   DateTime  @default(now())
```

But the `verificationTokenValidUntil` date uses my machine local date to generate the date instead of UTC. For example, I created a object at 12PM local time, and I am in the +2 timezone. Value of `createdAt` is `2022-06-12 10:25:52.309`, meanwhile the value of `verificationTokenValidUntil` is `2022-06-12 13:25:52.31`. Am I doing something wrong or is this a bug?

**Comment by austincrim** [comment]
Hey @randomprogramming,

This seems like a bug. Could you open a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with more info?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
